integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoQueryTestTool.java

```diff
@@ -0,0 +1,180 @@
+import java.util.Arrays;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.PosixParser;
+import org.apache.logging.log4j.util.Strings;
+
+public class TrinoQueryTestTool {
+
+  public static void main(String[] args) throws Exception {
+    Options options = new Options();
+
+    try {
+      options.addOption(
+          "auto",
+          true,
+          "Start the test containers and gravitino server automatically, the default value is 'all'. "
+              + "If the value is 'gravitino', only the gravitino server will be started automatically.");
+
+      options.addOption(
+          "gen_output",
+          false,
+          "Generate the output file for the test set, the default value is 'false'");
+
+      options.addOption(
+          "trino_uri", true, "URL for Trino, if --auto is set to 'all', this option is ignored");
+      options.addOption(
+          "hive_uri", true, "URL for Hive, if --auto is set to 'all', this option is ignored");
+      options.addOption(
+          "mysql_uri", true, "URL for MySQL, if --auto is set to 'all', this option is ignored");
+      options.addOption(
+          "hdfs_uri", true, "URL for HDFS, if --auto is set to 'all', this option is ignored");
+      options.addOption(
+          "postgresql_uri",
+          true,
+          "URL for PostgreSQL, if --auto is set to 'all', this option is ignored");
+
+      options.addOption(
+          "test_sets_dir",
+          true,
+          "Specify the test sets' directory, "
+              + "the default value is 'integration-test/src/test/resources/trino-queries'");
+      options.addOption("test_set", true, "Specify the test set name to test");
+      options.addOption("tester_id", true, "Specify the tester name prefix to select to test");
+      options.addOption("catalog", true, "Specify the catalog name to test");
+
+      options.addOption("help", false, "Print this help message");
+
+      CommandLineParser parser = new PosixParser();
+      CommandLine commandLine = parser.parse(options, args);
+
+      if (commandLine.hasOption("help")) {
+        HelpFormatter formatter = new HelpFormatter();
+        formatter.printHelp("TrinoTestTool", options);
+        return;
+      }
+
+      boolean autoStart = true;
+      boolean autoStartGravitino = true;
+      if (commandLine.getOptionValue("auto") != null) {
+        String auto = commandLine.getOptionValue("auto");
+        if (auto.equals("all")) {
+          autoStart = true;
+          autoStartGravitino = true;
+        } else if (auto.equals("gravitino")) {
+          autoStart = false;
+          autoStartGravitino = true;
+        } else if (auto.equals("none")) {
+          autoStart = false;
+          autoStartGravitino = false;
+        } else {
+          System.out.println("The value of --auto must be 'all' or 'gravitino'");
```

Comment:
If the value can only be `all` or `gravitino`, Why don't you stop the program directly when the value is `nono`

Reply:
I hope the default behavior is automatically start the test environment automatically such as docker containers and the Gravitino server.